### DISCUSSION TITLE
Ability to keep storyboard in beatmap when editing.

### DISCUSSION THREAD

**Discussion by SealsarecuteR** [ppy/osu#20881] [Ideas] [answered]
Right now, when you edit a beatmap, doing anything in the editor like adjusting the settings, or changing/modifying notes etc. will delete the storyboard, so you can't save the new version without the storyboard being gone. It would nice to be able to change a beatmap in lazer and have the storyboard still there. Not talking about the entire editor, just being able to save with it there.

**Comment by frenzibyte** [comment, marked as answer]
#21079
